src/pages/Landing/index.tsx

```diff
@@ -117,7 +118,10 @@ export default () => {
             onChange={(value) => setNewUser({ ...newUser, isCool: value })}
             isChecked={newUser.isCool}
           />
-          <Button type='submit'>Submit</Button>
+          <SnazzyButton type="submit" color="blue" isSnazzy>
+            Submit
+          </SnazzyButton>
+
```

Comment:
Awesome, you imported the button and replaced the previous one! 
You do have a trailing blank line on line 124. 
Remove that.